corehq/apps/importer/const.py

```diff
@@ -1,2 +1,11 @@
 class LookupErrors:
     NotFound, MultipleResults = range(2)
+
+
+class ImportErrors:
+    InvalidOwnerName = 'Invalid Owner Name'
+    InvalidOwnerId = 'Invalid Owner ID'
+    InvalidParentId = 'Invalid Parent ID'
+    InvalidDate = 'Invalid Date'
+    BlankExternalId = 'Blank External ID'
+    CaseGeneration = 'Case Generation Error'
```

Comment:
It's translated here: https://github.com/dimagi/commcare-hq/pull/5554/files#diff-aade6bc0f6a5a9e3de32a4322a87fafdR238

I thought it'd be weird if you were debugging on a different locale and the errors were in a different language. Not exactly sure how to best deal with this; do you know best practice?